circle.yml

```diff
@@ -13,6 +13,3 @@ jobs:
       - run:
           name: "Go mod tidy"
           command: ./hack/check-tidy.sh
-      - run:
```

Comment:
Remove .travis ci please, because we can merge the pr regardless of the travis ci failed.

Reply:
done